### DISCUSSION TITLE
Tangent Open Source Kickoff

### DISCUSSION THREAD

**Discussion by taylorhadden** [suchnsuch/Tangent#18] [Announcements]
# 👋 Welcome!
  Discussions will be one of the primary locations for discussing Tangent. We will use it to:
  * Ask and answer questions.
  * Share ideas and develop feature concepts.
  * Have a public and indexed location for our community's collective corpus.

Tangent has been in development since April of 2021. I've opened it up as an open source project for a couple of reasons.

One of the big ones is that, by and large, I have achieved what I set out to do when I started building Tangent. I have created a reliable "integrated thinking environment" that I use nearly every day, both for personal and professional work. That "ITE" has been explicitly tailored to myself, and am very comfortable within it. While that work is complete, Tangent has the potential to be so much more than what it is now. I have plenty of ideas, but I know other people do too. The project needs to expand as Tangent becomes more things for more people.

I'm not going to be able to provide that development bandwidth all by myself, and there's basically no chance of a traditional commercial enterprise working out the way any of use would like. With luck, an open source Tangent can continue to be a useful tool, and perhaps become a force to be reckoned with.

**Comment by david-aminpriddle** [comment]
## Created Issues

I've created GitHub issues for each feedback point:

### Signup Flow
- #751 - Show example data indicator in signup flow preview
- #752 - Update user avatar in signup preview when selected
- #753 - Fix empty avatar tooltip in signup flow
- #754 - Restrict company handle to alphanumeric and hyphens only
- #755 - Show company logo in signup preview
- #756 - Remove navigation from signup flow
- #757 - Implement back navigation between signup stages
- #758 - Update company signup copy to match prototype
- #759 - Add Volute logo to signup page

### Authentication
- #760 - Add logo branding to magic link verification page

### Admin
- #761 - Fix Admin/Users showing all users across organizations
- #762 - Investigate and fix crash during join request email verification
- #763 - Fix missing scrollbar in join request dialog on mobile

Each issue includes technical details, file references, and implementation guidance based on codebase analysis.

**Reply by taylorhadden** [reply]
HI David. You appear to have posted this in the wrong location!
